test/sqerl_rec_tests.erl

```diff
@@ -157,6 +157,11 @@ kitchen_test_() ->
                               "query did not return a single column",
                               [kitchen, fetch_all, []]}}},
                             sqerl_rec:scalar_fetch(kitchen, fetch_all, []))
+       end},
+      {"cinsert",
+       fun() ->
+               {K0, _Name} = make_kitchen(<<"pingpang">>),
+               {ok, 1} = sqerl_rec:cinsert(K0)
```

Comment:
Let's make that the convention with the reason being that what you'll get on failure has more detail and it also makes super clear what the primary assertion is in the test.